### PR TITLE
tikv: add doc for new config `max-apply-unpersisted-log-limit`

### PR DESCRIPTION
**Pull request by glorv**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add description for new config `raftstore.max-apply-unpersisted-log-limit`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [ ] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -739,6 +739,16 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.1.0 版本开始引入</span>
+
++ 允许 apply 已经 `commit` 但还没有持久化的 Raft日志的数量。默认情况下，TiKV 需要等待日志 commit 并且在本节点持久话之后才进行 apply, 当出现 IO 抖动时，会导致此节点的写入耗时大幅增加。将此配置设置为大于 0 的值可以有效的降低 IO 的抖动造成的长尾延迟增加。如果要开启此特性，建议设置为 1024 左右。
```
Comment (Connor1996) [inline tikv-configuration-file.md:744]:
```suggestion
+ 允许 apply 已经 `commit` 但还没有持久化的 Raft 日志的数量。默认情况下，TiKV 需要等待日志 commit 并且在本节点持久化之后才进行 apply, 当出现 IO 抖动时，会导致此节点的写入耗时大幅增加。将此配置设置为大于 0 的值时，会提前 apply 本地未持久化但已经 commit 的 Raft 日志，可以有效的降低 IO 的抖动造成的长尾延迟增加。如果要开启此特性，建议设置为 1024 左右。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -739,6 +739,12 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.2.0 版本开始引入</span>
+
++ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。将此配置设置为大于 0 的值将使该节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，可以有效降低少数 TiKV 节点的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft Engine 的 Raft Log 占用磁盘容量的增加。将此配置设置为 0 则表示关闭此特性，此时 TiKV 需要等待日志 commit 并且在本节点持久化之后才进行 apply，此行为和旧版本一致。
```
Comment (qiancai) [inline tikv-configuration-file.md:744]:
```suggestion
+ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Review by Connor1996 — APPROVED** [review]
LGTM

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -739,6 +739,12 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.2.0 版本开始引入</span>
+
++ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。
```
Comment (qiancai) [inline tikv-configuration-file.md:744]:
```suggestion
+ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。

	+ 将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。
	+ 将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。

```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -739,6 +739,16 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.2.0 版本开始引入</span>
+
++ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。
+
+	+ 将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。
+	+ 将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。
```
Comment (qiancai) [inline tikv-configuration-file.md:747]:
```suggestion
    + 将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。
    + 将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -739,6 +739,16 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.2.0 版本开始引入</span>
+
++ 允许 apply 已经 `commit` 但尚未持久化的 Raft 日志的最大数量。
+
+    + 将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上的 IO 抖动导致的长尾延迟，但也可能导致 TiKV 内存使用量的增加和 Raft 日志占用磁盘容量的增加。
+    + 将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为和 v8.2.0 之前版本的行为一致。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:747]:
```suggestion
    + 将此配置项设置为大于 0 的值将使该 TiKV 节点能够提前 apply 已 `commit` 但尚未持久化的 Raft 日志，从而有效降低该节点上因 IO 抖动导致的长尾延迟。但这也可能会增加 TiKV 内存使用量和 Raft 日志占用磁盘容量。
    + 将此配置项设置为 0 则表示关闭此特性，此时 TiKV 需要等待 Raft 日志被 `commit` 且持久化之后才能对其进行 apply，此行为与 v8.2.0 之前版本的行为一致。
```

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

File: tikv-configuration-file.md [hunk 6]
```diff
@@ -739,6 +739,16 @@ raftstore 相关的配置项。
 + 默认值：30s
 + 最小值：0
 
+### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.2.0 版本开始引入</span>
```
Comment (qiancai) [inline tikv-configuration-file.md:742]:
```suggestion
### `max-apply-unpersisted-log-limit` <span class="version-mark">从 v8.1.0 版本开始引入</span>
```

**Pull request merged**